Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -92,10 +92,12 @@ def runSelected(opt):
                      11634.911, #2021 DD4hep ttbar reading geometry from XML
                      11634.914, #2021 DDD ttbar reading geometry from the DB
                      11634.0, #2021 ttbar (switching to DD4hep by default)
+                     11723.17, #2021 QCD Pt_1800_2400 seedingDeepCore; Temporary
                      13234.0, #2021 ttbar fastsim
                      12434.0, #2023 ttbar
                      12634.0, #2023 ttbar PU
                      12434.7, #2023 ttbar mkFit
+                     12434.701, #2023 ttbar displacedRegional; Temporary
```

Comment:
IIUC, in order to make this available by default (i.e. without specifying `--what upgrade`) in the limited matrix it needs to appear in this list as well:

https://github.com/cms-sw/cmssw/blob/27bec5ff620bde58ac4a2a2812a545e881c47117/Configuration/PyReleaseValidation/python/relval_2017.py#L50

Reply:
That is correct. Please add it to the list.